activemerchant.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
   s.has_rdoc = true if Gem::VERSION < '1.7.0'
 
-  s.add_dependency('activesupport', '>= 3.2.14', '< 6.0')
+  s.add_dependency('activesupport', '>= 3.2.14', '< 7.0')
```

How about:
```suggestion
  s.add_dependency('activesupport', '~> 6.0')
```

Or, if you're feeling lucky:
```suggestion
  s.add_dependency('activesupport', '~> 6')
```

---

Unfortunately, we can't do that now because main app is still on Rails < 6.0